python/Ganga/test/GPI/FileTests/TestMassStorageWN.py

```diff
@@ -0,0 +1,249 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+import tempfile
+import datetime
+import time
+import random
+import os
+import shutil
+import copy
+import string
+
+def generateUniqueTempFile( ext = '.txt' ):
```

Comment:
I would have preferred a common file in this dir but `testlib` could be used if it's going to be used in other tests too.